csharp/Samples/Microsoft.Spark.CSharp/DStreamSamples.cs

```diff
@@ -83,6 +88,9 @@ internal static void DStreamTextFileSamples()
                 System.Threading.Thread.Sleep(1000);
             }
 
+            // wait ForeachRDD to complete to let ssc.Stop() gracefully
+            System.Threading.Thread.Sleep(2000);
```

Comment:
Are we recommending to follow the same practice when they write their streaming apps in SparkCLR?